flocker/node/agents/cinder.py

```diff
@@ -9,26 +9,33 @@
 
 from bitmath import Byte, GiB
 
-from eliot import Message, MessageType, Field, start_action
+from eliot import Message, start_action
 
 from pyrsistent import PRecord, field
 
 from keystoneclient.openstack.common.apiclient.exceptions import (
     NotFound as CinderNotFound,
+    HttpError as KeystoneHttpError,
 )
 from novaclient.exceptions import NotFound as NovaNotFound
+from novaclient.exceptions import ClientException as NovaClientException
 
 from twisted.python.filepath import FilePath
 
 from zope.interface import implementer, Interface
 
 from ...common import (
-    auto_openstack_logging, get_all_ips, ipaddress_from_string
+    interface_decorator, get_all_ips, ipaddress_from_string
+    # auto_openstack_logging, get_all_ips, ipaddress_from_string
```

Comment:
Yes, fixed. Left out cleanup in this draft to allow undo in case i was going in the wrong direction.